File: VB/GettingStarted2/Controllers/ChartController.vb
Imports System.Web.Mvc

Namespace Controllers
    Public Class ChartController
        Inherits Controller

        Function Index() As ActionResult
            Using context As New NwindDbContext
                Dim beveragesCategory As Category
                beveragesCategory = context.Categories.Where(Function(c) c.CategoryName = "Beverages").Single()
                Return View(beveragesCategory.Products.ToList())
            End Using
        End Function
    End Class
End Namespace

File: CS/GettingStarted2/Controllers/ChartController.cs
using GettingStarted2.Models;
using System.Linq;
using System.Web.Mvc;

namespace GettingStarted2.Controllers {
    public class ChartController : Controller {
        public ActionResult Index() {
            using(var context = new NwindDbContext()) {
                Category beveragesCategory = context.Categories
                        .Where(c => c.CategoryName == "Beverages")
                        .Single();
                return View(beveragesCategory.Products.ToList());
            }
        }
    }
}